firmware/thruna/DIM/src/cubemx/DIM.ioc.md5

```diff
@@ -1 +1 @@
-d41ad53b247e4e1d6391ac3ac07808ba
```

Comment:
Looking at my branch right now, I think I made some branch errors, will fix for tomorrow. I'll try to get a clean PR based on master rather than Amir's branch, sorry for the md5 mess.

Same thing applies to vehicle dynamics.